libvolk.lwr

```diff
@@ -19,11 +19,11 @@
 
 category: common
 depends:
+- cheetah
```

Comment:
`cheetah` is only required for old releases. It was replaced by Mako because cheetah is abandoned and does not support Python3. Thus, this line will most likely trigger a complete Python2 install on a modern system. I'd argue that is an unwanted side-effect.
Mako, and previously Cheetah, are build time only dependencies. I'd argue that if you need an older release, please use whatever your package manager provides. In that case, you need neither Mako nor Cheetah installed.